docker/README.md

```diff
@@ -15,11 +15,12 @@ docker build -t r2l .
 
 ```shell
 cd ~/ros2learn/docker
-docker rm r2l || true && docker run -it --name=r2l -h ros2learn -v `pwd`/tmp:/tmp r2l
+docker rm r2l || true && docker run -it --name=r2l -h ros2learn -v `pwd`/tmp:/tmp/ros2learn r2l
 
 #Inside the docker container, used to load visual models
 cp -r /root/ros2_mara_ws /tmp
```

Comment:
Shouldn't this line be `cp -r /root/ros2_mara_ws tmp/ros2learn` ?

Reply:
Yes!

Reply:
Indeed we needed this. Commited the change.